odoo/tests/common.py

```diff
@@ -537,8 +537,24 @@ def executable(self):
 
         raise unittest.SkipTest("Chrome executable not found")
 
+    def _spawn_chrome(self, cmd):
+        pid = os.fork()
+        if pid != 0:
+            return pid
+        else:
+            if os.name == 'posix' and platform.system() != 'Darwin':
+                # since the introduction of pointer compression in Chrome 80 (v8 v8.0),
+                # the memory reservation algorithm requires more than 8GiB of virtual mem for alignment
+                # this exceeds our default memory limits.
+                # OSX already reserve huge memory for processes
+                resource.setrlimit(resource.RLIMIT_AS, (resource.RLIM_INFINITY, resource.RLIM_INFINITY))
+                stdout_replacement = os.open(os.devnull, os.O_WRONLY, 0)
+                os.dup2(stdout_replacement, sys.stderr.fileno())
+                os.close(stdout_replacement)
```

Comment:
A slightly cleaner version might be
```python
with contextlib.closing(os.open(os.devnull, ...)) as stdout_replacement:
    os.dup2(stdout_replacement, sys.stderr.fileno())
```
or
```python
with open(os.devnull, 'w') as stdout_replament:
    os.dup2(stdout_replacement.fileno(), sys.stderr.fileno())
```

Reply:
I agree and prefer the second one